### PR TITLE
import This Week in Rails from 04.03

### PR DESCRIPTION
**Pull request by morgoth**
/cc @chancancode @vipulnsward

### CODE REVIEW

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Featured
+
+### [This week's Rails contributors](http://contributors.rubyonrails.org/contributors/in-time-window/20160227-20160304)
+
+This week 48 contributors helped to move Rails forward! Want to be one of them? Look at the [issues](https://github.com/rails/rails/issues) list and make a contribution!
+
+### [Rails 4.2.6.rc1 and 4.1.15.rc1 have been released](http://weblog.rubyonrails.org/2016/3/1/Rails-4-2-6-rc1-and-4-1-15-rc1-have-been-released/)
+
+Rails 4.2.6.rc1 and 4.1.15.rc1 have been released!
+If no regressions are found expect the final release around March 4, 2016.
```
Comment (connorshea) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: It's March 6, 2016, so I'm guessing that didn't pan out?

Comment (morgoth) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: @rafaelfranca wanted to do it, but looks like something stopped him.

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    Rails.application.reloader.wrap do
+      # run some user code
+    end
+
+and Rails will take care of the interlock, code reloading, returning `ActiveRecord` connections to the pool, and anything else that might be relevant.
+Exciting, if you have ever been stung by concurrent processing in Jobs, etc.
+
+## New Stuff
+
+### [Rails now depends on Puma 3](https://github.com/rails/rails/pull/23906)
```
Comment (vipulnsward) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: Can you change this to defaults

Comment (morgoth) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: @vipulnsward What do you mean?

**Comment by chancancode** [comment]
We forgot the chance the subject line before we send the email (it has the same subject/title as last time), we should probably fix it in the blog :smile: @vipulnsward you wanna pick a title?

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown (hunk 2 above)
Comment (vipulnsward) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: s/depends/defaults/

**Comment by chancancode** [comment]
(Should update the filename too since it's used for the permalink)

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown (hunk 2 above)
Comment (morgoth) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: done `depends on` -> `defeaults to`

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "This week in Rails: Turbolinks 5, Rails 5.0.0.beta3 and more!"
```
Comment (vipulnsward) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: "This week in Rails: Rails 4.2.6.rc1 and 4.1.15.rc1, AS::Executor and AS::Reloader APIs and more!"

File: _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+These both allow external code to run blocks of user code to do "work" at a similar unit size to a web request without needing to get intimate with `ActionDispatch`.
+
+This change is intended to allow interested callers (`Sidekiq`, `ActionCable`), to just do:
+
+    Rails.application.reloader.wrap do
+      # run some user code
+    end
+
+and Rails will take care of the interlock, code reloading, returning `ActiveRecord` connections to the pool, and anything else that might be relevant.
```
Comment (vipulnsward) [inline _posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown]: @fxn I believe you mean't all mentions of components here need to be changed. `ActiveRecord` => Active Record.
I pulled this mostly from https://github.com/rails/rails/pull/23807

**Comment by morgoth** [comment]
I updated title and file name

**Comment by vipulnsward** [comment]
Thanks, needs one confirmation and change, then ready to go!

**Comment by morgoth** [comment]
Guys, I go to sleep :smile:  - please merge it locally and do some further edits if needed.

**Comment by morgoth** [comment]
Any news? It will get outdated pretty soon if we don't publish it

**Comment by fxn** [comment]
Did a quick read:
- The name of Rails components goes in regular font and has a space. For example `ActionDispatch` would be Action Dispatch. Same for Action Cable and others mentioned in the post.
- `Sidekiq` is the name of a project. Like Ruby on Rails or MySQL, goes in regular font.
- > Exciting, if you have ever been stung by concurrent processing in Jobs
  
  No need to use a capital "J" in "Jobs".
- In our guidelines file names go in fixed-width font, so `config/puma.rb` for example.
- Fixed-width font for `force_ssl`.
- "https" would be "HTTPS", that is the way to spell the protocol.
- Spurious hyphen in "for example-".
- "trying to blacklist or sanitize `bad` values" no need to put "bad" in fixed-width font.
- `DatabaseStatements` needs a full contant path. It doesn't exist like that in `Object`.
- "ruby 2.2+" Ruby, the language, is written with a capital "R".

**Comment by morgoth** [comment]
@fxn Thanks, I did the corrections that you pointed.

**Comment by fxn** [comment]
There is an extra space in:

> when booting from the command `$ rails server` .

Then, in "application's need to reload App changes", "App" written that way and with a capital "A" makes no sense. This sentence could be just shortened: "This file can then be edited to suit each application" or something simple like that. It could even be deleted, since this is something you take for granted in any of those files.

With those edits I am good, leaving to @chancancode the final thumbs up.

**Comment by chancancode** [comment]
:clap: thank you all!

**Pull request merged**